src/components/EventFilterBar.js

```diff
@@ -47,12 +47,19 @@ const EventFilterBar = ({
             <div className="filter-bar__browse--menu">
               <div className="filter-bar__browse--title-wrapper">
                 <span className="filter-bar__browse--title">Browse By</span>
-                <img
-                  src={down}
-                  alt=""
-                  className="filter-bar__browse--arrow"
-                  onClick={() => setToggle(!isToggle)}
-                />
+                <div className="filter-bar__browse--arrow-wrapper">
+                  <img
+                    src={down}
+                    alt=""
+                    className="filter-bar__browse--arrow"
+                    onClick={() => setToggle(!isToggle)}
```

Comment:
I think the text should also be clickable as well: 
![image](https://user-images.githubusercontent.com/9009552/82372404-f0868980-99e9-11ea-9125-24867cf159a5.png)

Reply:
Sorry, I realize that's not exactly new in this PR